beanie/odm/documents.py

```diff
@@ -691,13 +691,13 @@ async def replace_many(
     @save_state_after
     async def update(
         self: Self,
-        *args,
+        *args: Any,
```

Comment:
Not aligned with the docstring below:
`:param args: *Union[dict, Mapping] - the modifications to apply.`
And docstring is generally out of date as it doesn't mention all new params.